### PR TITLE
769: Carousel article filters

### PR DESCRIPTION
**Pull request by rimi-itk**
#### Link to ticket

https://leantime.itkdev.dk/#/tickets/showTicket/769

#### Description

Adds article carousel and list filters based on article type and subjects.

#### Checklist

- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

**Review by cableman — APPROVED** [review]

File: sites/all/modules/ereol_article/ereol_article.views_default.inc [hunk 1]
```diff
@@ -84,23 +85,51 @@ function ereol_article_views_default_views() {
   $handler->display->display_options['exposed_form']['options']['exposed_sorts_label'] = 'Sortér efter';
   $handler->display->display_options['exposed_form']['options']['sort_asc_label'] = 'Stigende';
   $handler->display->display_options['exposed_form']['options']['sort_desc_label'] = 'Faldende';
-  $handler->display->display_options['exposed_form']['options']['autosubmit'] = TRUE;
   $handler->display->display_options['exposed_form']['options']['bef'] = array(
     'general' => array(
+      'input_required' => 0,
+      'text_input_required' => array(
+        'text_input_required' => array(
+          'value' => 'Vælg et filter og klik på Anvend for at se resultater',
```
Comment (cableman) [inline sites/all/modules/ereol_article/ereol_article.views_default.inc:93]: Should this be English?

File: sites/all/modules/ereol_article/ereol_article.views_default.inc [hunk 2]
```diff
@@ -153,8 +209,11 @@ function ereol_article_views_default_views() {
     t('sidste »'),
     t('Side'),
     t('more'),
-    t('Select any filter and click on Apply to see results'),
+    t('Vælg et filter og klik på Anvend for at se resultater'),
+    t('Avancerede indstillinger'),
```
Comment (cableman) [inline sites/all/modules/ereol_article/ereol_article.views_default.inc:213]: Danish to danish translations?

**Review by cableman — CHANGES_REQUESTED** [review]

File: sites/all/modules/ereol_article/ereol_article.views_default.inc (hunk 2 above)
Comment (rimi-itk) [reply sites/all/modules/ereol_article/ereol_article.views_default.inc:213]: That depends on the source language. I have no idea why `drush features-update` generates source strings in Danish, but “Side” and “sidste »" above this is also in Danish.

File: sites/all/modules/ereol_article/ereol_article.views_default.inc (hunk 1 above)
Comment (rimi-itk) [reply sites/all/modules/ereol_article/ereol_article.views_default.inc:93]: I have no idea why `drush features-update` generates texts in Danish. I could manually change it, but I risk breaking more than I'm fixing.

File: sites/all/modules/ereol_article/ereol_article.views_default.inc (hunk 2 above)
Comment (rimi-itk) [reply sites/all/modules/ereol_article/ereol_article.views_default.inc]: Texts translated to English in https://github.com/eReolen/base/pull/345/commits/c2448f3223de646afbf214307b95cc2357f18ea1.

**Review by cableman — APPROVED** [review]

**Review by cableman — APPROVED** [review]

**Review by cableman — APPROVED** [review]

**Pull request merged**